Linkinlog/HomeBurrow
# Backend/create defaults

## What is getting changed with this PR
- Removed the bash script that ran on startup for the db container
- Replaced it with a sql file that will make a few default tables
- Updated the readme

## Why are we making these changes?
- So we can have a first version of a standard DB structure 

## Outline testing instructions
In order to test this you will need to remove the docker volume `homeburrow-db` and the docker image `mariadb`. I added a make command into the makefile so we can do this easily.
- Run the make command listed below (more info about the make command can be found in the readme, but not really important)
- Run the test command below and ensure it passes
- (Optional) Use a DB client and confirm the tables show up

## Commands to run
```BASH
make duke_nukem
make test
```
## Checklist before requesting a review
- [x] If it is a large change, I have added thorough unit tests if applicable.
- [x] The testing instructions accurately explain how to confirm that this code change works
- [x] I have requested a review from at least 1 other person
